internal/attestation/measurements/measurements.go

```diff
@@ -153,7 +148,7 @@ func (m *M) FetchAndVerify(
 // The hash of the fetched measurements is returned.
 func (m *M) fetchAndVerify(
```

Comment:
should we use the new api client here too?

Reply:
I would like to migrate that use as well, but I think this may be more involved. I am currently focused on migrating Azure SNP and creating the AWS objects.